rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -816,6 +816,13 @@ void getRetiringFederationThreshold_returnsRetiringFederationNonExistentResponse
             int retiringFederationThreshold = federationSupport.getRetiringFederationThreshold();
             assertThat(retiringFederationThreshold, is(FederationChangeResponseCode.RETIRING_FEDERATION_NON_EXISTENT.getCode()));
         }
+
+        @Test
+        @Tag("getRetiringFederationCreationTime")
+        void getRetiringFederationCreationTime_returnsNull() {
+            Instant retiringFederationCreationTime = federationSupport.getRetiringFederationCreationTime();
```

Comment:
This might be a good chance to make the method return an `Optional` instead of null. Anyways, LGTM!

Reply:
Yeap, we have that in another task